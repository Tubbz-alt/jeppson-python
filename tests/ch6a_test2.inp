    6    5    3  100  5.0E-4  
    1    2   4.0    130.0     150.0     
    1    4   3.0    130.0     100.0     
    2    3   3.0    130.0     100.0     
    2    5   4.0    130.0     100.0     
    3    5   3.0    130.0     141.0     
    4    3   3.0    130.0     150.0     
    1   1.0     115.0    
    2   0.0     103.0    
    3  -0.25    100.0    
    4  -0.25    103.0    
    5  -0.5      99.0    
