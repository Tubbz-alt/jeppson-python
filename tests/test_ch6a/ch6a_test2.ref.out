Pipe    Diameter         Length         CHW
  0     4.0000E+00 in    1.5000E+02 ft  130.000
  1     3.0000E+00 in    1.0000E+02 ft  130.000
  2     3.0000E+00 in    1.0000E+02 ft  130.000
  3     4.0000E+00 in    1.0000E+02 ft  130.000
  4     3.0000E+00 in    1.4100E+02 ft  130.000
  5     3.0000E+00 in    1.5000E+02 ft  130.000


Iteration 0
Deviation 8.8526E+00 (Tolerance 5.0000E-04

Pipe   Kp            Head loss        Volumetric Flow
  0    1.8197E+01    7.0884E+00 ft    1.4796E+00 ft ** 3 / s
  1    4.9250E+01    7.7776E+00 ft    9.4830E-01 ft ** 3 / s
  2    4.9250E+01    2.1408E+00 ft    2.6102E-01 ft ** 3 / s
  3    1.2131E+01    2.4667E+00 ft    6.4092E-01 ft ** 3 / s
  4    6.9443E+01    3.2588E-01 ft    3.3005E-02 ft ** 3 / s
  5    7.3875E+01    1.4516E+00 ft    1.4218E-01 ft ** 3 / s

Junction    Head
  0         1.0923E+02 ft
  1         1.0214E+02 ft
  2         1.0000E+02 ft
  3         1.0145E+02 ft
  4         9.9674E+01 ft

Iteration 1
Deviation 8.3129E-01 (Tolerance 5.0000E-04

Pipe   Kp            Head loss        Volumetric Flow
  0    1.8197E+01    7.5530E+00 ft    1.5766E+00 ft ** 3 / s
  1    4.9250E+01    8.1126E+00 ft    9.8915E-01 ft ** 3 / s
  2    4.9250E+01    2.1887E+00 ft    2.6686E-01 ft ** 3 / s
  3    1.2131E+01    2.6079E+00 ft    6.7761E-01 ft ** 3 / s
  4    6.9443E+01    4.1922E-01 ft    4.2459E-02 ft ** 3 / s
  5    7.3875E+01    1.6291E+00 ft    1.5957E-01 ft ** 3 / s

Junction    Head
  0         1.0974E+02 ft
  1         1.0219E+02 ft
  2         1.0000E+02 ft
  3         1.0163E+02 ft
  4         9.9581E+01 ft

Iteration 2
Deviation 1.6601E-02 (Tolerance 5.0000E-04

Pipe   Kp            Head loss        Volumetric Flow
  0    1.8197E+01    7.5608E+00 ft    1.5782E+00 ft ** 3 / s
  1    4.9250E+01    8.1146E+00 ft    9.8939E-01 ft ** 3 / s
  2    4.9250E+01    2.1872E+00 ft    2.6667E-01 ft ** 3 / s
  3    1.2131E+01    2.6110E+00 ft    6.7843E-01 ft ** 3 / s
  4    6.9443E+01    4.2389E-01 ft    4.2932E-02 ft ** 3 / s
  5    7.3875E+01    1.6333E+00 ft    1.5998E-01 ft ** 3 / s

Junction    Head
  0         1.0975E+02 ft
  1         1.0219E+02 ft
  2         1.0000E+02 ft
  3         1.0163E+02 ft
  4         9.9576E+01 ft

Iteration 3
Deviation 1.5382E-05 (Tolerance 5.0000E-04

Pipe   Kp            Head loss        Volumetric Flow
  0    1.8197E+01    7.5608E+00 ft    1.5782E+00 ft ** 3 / s
  1    4.9250E+01    8.1146E+00 ft    9.8939E-01 ft ** 3 / s
  2    4.9250E+01    2.1871E+00 ft    2.6667E-01 ft ** 3 / s
  3    1.2131E+01    2.6111E+00 ft    6.7843E-01 ft ** 3 / s
  4    6.9443E+01    4.2390E-01 ft    4.2933E-02 ft ** 3 / s
  5    7.3875E+01    1.6333E+00 ft    1.5998E-01 ft ** 3 / s

Junction    Head
  0         1.0975E+02 ft
  1         1.0219E+02 ft
  2         1.0000E+02 ft
  3         1.0163E+02 ft
  4         9.9576E+01 ft

Final results:

  Pipe  From    To  Diameter         Length         CHW      Volumetric Flow    Head Loss      From Head      To Head
  0     0       1   4.0000E+00 in    1.5000E+02 ft  130.000  1.578 ft ** 3 / s  7.5608E+00 ft  1.0975E+02 ft  1.0219E+02 ft
  1     0       3   3.0000E+00 in    1.0000E+02 ft  130.000  0.989 ft ** 3 / s  8.1146E+00 ft  1.0975E+02 ft  1.0163E+02 ft
  2     1       2   3.0000E+00 in    1.0000E+02 ft  130.000  0.267 ft ** 3 / s  2.1871E+00 ft  1.0219E+02 ft  1.0000E+02 ft
  3     1       4   4.0000E+00 in    1.0000E+02 ft  130.000  0.678 ft ** 3 / s  2.6111E+00 ft  1.0219E+02 ft  9.9576E+01 ft
  4     2       4   3.0000E+00 in    1.4100E+02 ft  130.000  0.043 ft ** 3 / s  4.2390E-01 ft  1.0000E+02 ft  9.9576E+01 ft
  5     3       2   3.0000E+00 in    1.5000E+02 ft  130.000  0.160 ft ** 3 / s  1.6333E+00 ft  1.0163E+02 ft  1.0000E+02 ft


Junction  Inflow                  Head
  0       1.0000E+00 ft ** 3 / s  1.1500E+02 ft  
  1       0.0000E+00 ft ** 3 / s  1.0300E+02 ft  
  2       -2.5000E-01 ft ** 3 / s  1.0000E+02 ft  Fixed head
  3       -2.5000E-01 ft ** 3 / s  1.0300E+02 ft  
  4       -5.0000E-01 ft ** 3 / s  9.9000E+01 ft  

